app/javascript/hooks/queries/rooms/useRoomJoin.jsx

```diff
@@ -10,7 +10,7 @@ export default function useRoomJoin(friendlyId, name, accessCode) {
 
   return useQuery(
     ['getRoomJoin', name],
-    () => axios.get(`/meetings/${friendlyId}/join.json`, { params }).then((resp) => { subscribeToRoom(friendlyId, resp.data.data); }),
+    () => axios.get(`/meetings/${friendlyId}/join.json`, { params }).then((resp) => { subscribeToRoom(friendlyId, resp.data); }),
```

Comment:
Shouldn't this introduce a bug after merging #3707 since it will be encapsulated inside of `data`?

Reply:
Probably - was going to retest it when it's merged

Reply:
Makes sense,
I suspect either that the start may fail either because it's manually encapsulating the data:
```
       # MeetingsCotroller#start
       ###################
        ...
        render_data data: {
          join_url: BigBlueButtonApi.new.join_meeting(room: @room, name: current_user.name, role: 'Moderator')
        }, status: :created
```
Because it's a `Hash` it should be wrapped manually inside of a `data` property, that will lead to having an extra encapsulation (not sure) but just to be aware of it.